tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "outDir": "dist",
     "target": "es2018",
-    "lib": ["es2018"],
+    "lib": ["es2018", "ES2019.Array"],
```

Comment:
What does this provide?

Reply:
This snuck in early but is for using `array.flat()` in #3